todo_app/adapters/mongodb_user.py

```diff
@@ -0,0 +1,82 @@
+import os, ssl, pymongo
+from bson.objectid import ObjectId
+
+class Connection():    
+    def __init__(self): 
+        self.mongo_connection_string = os.environ['MONGO_CONNECTION_STRING']
+        self.mongo_collection_name = os.environ['MONGODB_COLLECTIONNAME']
+        self.client = pymongo.MongoClient(self.mongo_connection_string, ssl_cert_reqs=ssl.CERT_NONE)
+        self.mongo_db = self.client.todo_app
+        self.collection = self.mongo_db.users
+        
+class mongodb_user(Connection):
```

Comment:
Small but this name is a little misleading - it suggests it is representing a particular user, but it's really a helper class to integrate with the user collection. Maybe `MongoDbUserService` would be clearer?

Reply:
TY. I will look at this as part of my Mod 12.